app/graphql/types/session.rb

```diff
@@ -16,4 +16,15 @@ def account
   def profile
     object unless object.blank?
   end
+
+  field :nolt_token, String,
+    null: false,
+    description: 'Single sign-on token for Nolt'
+
+  def nolt_token
+    user = context[:user]
+    raise GraphQL::ExecutionError, 'You must be logged in to do that' if user.blank?
```

Comment:
I feel like we should create an `auth_field` at some point that does this. Prob not this PR just wanted to bring the idea up.

Reply:
Yeah seems like a good idea